It is complaining about the use of accessing the package directly without annotation
```
java.lang.AssertionError: Architecture Violation [Priority: MEDIUM] - Rule 'no classes that are not annotated with @ApacheCommonsLang3Allowed should access classes that reside in a package 'org.apache.commons.lang3'' was violated (8 times):
```

Failed test
```
MainArchitectureTest. doNotUseApacheCommonsLang3
MainArchitectureTest. doNotUseJavaAWT
MainArchitectureTest. restrictStandardStreams
```

--- a/src/test/java/org/jabref/gui/ClipBoardManagerTest.java
+++ b/src/test/java/org/jabref/gui/ClipBoardManagerTest.java
@@ -2,6 +2,7 @@ package org.jabref.gui;
 
 import java.awt.Toolkit;
 import java.awt.datatransfer.StringSelection;
+import java.io.IOException;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
@@ -11,6 +12,7 @@ import javafx.collections.FXCollections;
 import javafx.collections.ObservableList;
 import javafx.scene.input.Clipboard;
 
+import org.jabref.architecture.AllowedToUseAwt;
 import org.jabref.logic.bibtex.FieldPreferences;
 import org.jabref.model.entry.BibEntry;
 import org.jabref.model.entry.BibEntryType;
@@ -30,6 +32,7 @@ import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
+@AllowedToUseAwt("Requires AWT for clipboard access")
 public class ClipBoardManagerTest {
 
     private BibEntryTypesManager entryTypesManager;
@@ -63,7 +66,7 @@ public class ClipBoardManagerTest {
 
     @DisplayName("Check that the ClipBoardManager can set a bibentry as its content from the clipboard")
     @Test
-    void testCopyStringBibEntry() {
+    void testCopyStringBibEntry() throws IOException {
         // Arrange
         String expected = "@Article{,\n author = {Claudepierre, S. G.},\n journal = {IEEE},\n}";
 
@@ -78,11 +81,7 @@ public class ClipBoardManagerTest {
         bibEntries.add(bibEntry);
 
         // Act
-        try {
-            clipBoardManager.setContent(bibEntries, entryTypesManager);
-        } catch (Exception e) {
-            e.printStackTrace(System.out);
-        }
+        clipBoardManager.setContent(bibEntries, entryTypesManager);
 
         // Assert
         String actual = ClipBoardManager.getContentsPrimary();
@@ -95,7 +94,7 @@ public class ClipBoardManagerTest {
 
     @Test
     @DisplayName("Check that the ClipBoardManager can handle a bibentry with string constants correctly from the clipboard")
-    void testCopyStringBibEntryWithStringConstants() {
+    void testCopyStringBibEntryWithStringConstants() throws  {
         // Arrange
         String expected = "@String{grl = \"Geophys. Res. Lett.\"}@Article{,\n" + " author = {Claudepierre, S. G.},\n" +
                 " journal = {grl},\n" + "}";
@@ -121,11 +120,7 @@ public class ClipBoardManagerTest {
         constants.add(bibtexString);
 
         // Act
-        try {
-            clipBoardManager.setContent(bibEntries, entryTypesManager, constants);
-        } catch (Exception e) {
-            e.printStackTrace(System.out);
-        }
+        clipBoardManager.setContent(bibEntries, entryTypesManager, constants);
 
         // Assert
         String actual = ClipBoardManager.getContentsPrimary();

--- a/src/test/java/org/jabref/gui/ClipBoardManagerTest.java
+++ b/src/test/java/org/jabref/gui/ClipBoardManagerTest.java
@@ -94,7 +94,7 @@ public class ClipBoardManagerTest {
 
     @Test
     @DisplayName("Check that the ClipBoardManager can handle a bibentry with string constants correctly from the clipboard")
-    void testCopyStringBibEntryWithStringConstants() throws  {
+    void testCopyStringBibEntryWithStringConstants() throws IOException {
         // Arrange
         String expected = "@String{grl = \"Geophys. Res. Lett.\"}@Article{,\n" + " author = {Claudepierre, S. G.},\n" +
                 " journal = {grl},\n" + "}";

--- a/src/test/java/org/jabref/gui/ClipBoardManagerTest.java
+++ b/src/test/java/org/jabref/gui/ClipBoardManagerTest.java
@@ -21,13 +21,11 @@ import org.jabref.model.entry.field.StandardField;
 import org.jabref.model.entry.types.StandardEntryType;
 import org.jabref.preferences.PreferencesService;
 
-import org.apache.commons.lang3.StringUtils;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotEquals;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
@@ -88,8 +86,11 @@ public class ClipBoardManagerTest {
 
         // Assert
         String actual = ClipBoardManager.getContentsPrimary();
-        assertNotEquals("test", StringUtils.normalizeSpace(actual));
-        assertEquals(StringUtils.normalizeSpace(expected), StringUtils.normalizeSpace(actual));
+        // clean strings
+        actual = actual.replaceAll("\\s+", " ").trim();
+        expected = expected.replaceAll("\\s+", " ").trim();
+
+        assertEquals(expected, actual);
     }
 
     @Test
@@ -128,7 +129,10 @@ public class ClipBoardManagerTest {
 
         // Assert
         String actual = ClipBoardManager.getContentsPrimary();
-        assertNotEquals("test", StringUtils.normalizeSpace(actual));
-        assertEquals(StringUtils.normalizeSpace(expected), StringUtils.normalizeSpace(actual));
+        // clean strings
+        actual = actual.replaceAll("\\s+", " ").trim();
+        expected = expected.replaceAll("\\s+", " ").trim();
+
+        assertEquals(expected, actual);
     }
 }

--- a/src/test/java/org/jabref/gui/libraryproperties/constants/ConstantsPropertiesViewModelTest.java
+++ b/src/test/java/org/jabref/gui/libraryproperties/constants/ConstantsPropertiesViewModelTest.java
@@ -11,7 +11,6 @@ import org.jabref.model.database.BibDatabaseContext;
 import org.jabref.model.entry.BibtexString;
 import org.jabref.preferences.FilePreferences;
 
-import org.apache.commons.lang3.StringUtils;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
@@ -127,6 +126,10 @@ class ConstantsPropertiesViewModelTest {
         List<String> actual = context.getDatabase().getStringValues().stream()
                                      .map(BibtexString::getParsedSerialization).toList();
 
-        assertEquals(StringUtils.normalizeSpace(expected.getFirst()), StringUtils.normalizeSpace(actual.getFirst()));
+        // get the first value and clean strings
+        String actual_value = actual.getFirst().replaceAll("\\s+", " ").trim();
+        String expected_value = expected.getFirst().replaceAll("\\s+", " ").trim();
+
+        assertEquals(expected_value, actual_value);
     }
 }

